aiida/cmdline/commands/cmd_daemon.py

```diff
@@ -76,7 +77,10 @@ def start(foreground, number):
 @verdi_daemon.command()
 @click.option('--all', 'all_profiles', is_flag=True, help='Show all daemons.')
 def status(all_profiles):
-    """Print the status of the current daemon or all daemons."""
+    """Print the status of the current daemon or all daemons.
+
+    Returns exit code 0 if all daemons are running, else exit code 3.
```

Comment:
Could you please clarify that "all daemons" refers to "all daemons whose status is requested" and not "daemons of all profiles", i.e., if one does not specify `--all` then 0 is returned when the current default daemon/profile is running, even if any other daemon is not running

Reply:
sure, done!